### PR TITLE
chore: Make renovate bot selective to manually approved upgrades from dashboard.

### PR DESCRIPTION
**Pull request by ddixit14**
This PR limits the scope of RenovateBot to only manually approved upgrades.

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -14,16 +14,24 @@
...
-      "packagePatterns": ["^com.google.protobuf:protobuf-"],
+      "matchPackagePatterns": ["^com.google.protobuf:protobuf-"],
       "groupName": "Protobuf packages"
     },
     {
-      "packagePatterns": ["^io.opencensus:opencensus-"],
+      "matchPackagePatterns": ["^io.opencensus:opencensus-"],
       "groupName": "OpenCensus packages"
-    }
+    }
```
Comment (suztomo) [inline renovate.json:36]: Remove unnecessary space

**Review by suztomo — APPROVED** [review]

**Comment by meltsufin** [comment]
@ddixit14 @suztomo What is the problem we're solving here?

**Comment by suztomo** [comment]
@meltsufin This https://github.com/googleapis/google-cloud-java/issues/8239

File: renovate.json (hunk 1 above)
Comment (suztomo) [reply renovate.json:27]: Remove unnecessary space

**Review by suztomo — APPROVED** [review]
Remove unnecessary space

**Review by meltsufin — APPROVED** [review]

**Pull request merged**